### PR TITLE
"Stacked" design on Add Site page

### LINKED ISSUE #1585: Implement new "stacked" design on Add Site page
#### Description of feature or bug

From @ericronne's [comment](https://github.com/18F/federalist/pull/1544#issuecomment-362610835):

image

#### Definition of done

- [x] design is implemented and ok'd by @ericronne
- [ ] template descriptions are adjusted to fit new design

_After evaluating, edit this part:_

#### Implementation outline (if higher than "low" effort):

<TODO>

### PR DESCRIPTION
**Pull request by line47**
### Initial WIP screenshot
screen shot 2018-02-02 at 12 12 44 pm

Haven't looked at tests yet... 

CC @ericronne 

closes #1585

### CODE REVIEW

**Comment by line47** [comment]
@ericronne have a look 👀  and see if there is anything you'd like to adjust...

### Screenshot
localhost_1337_sites_new

**Comment by ericronne** [comment]
_looksee in progress …_

**Comment by ericronne** [comment]
Vertical space between the template titles and their text (see below) should be reduced a little, otherwise looks great!

image

**Comment by line47** [comment]
Great! Pushing an update now!

**Comment by line47** [comment]
@ericronne - I'm on it 🎉

**Comment by line47** [comment]
I noticed that the form fields on the site settings page are not `3px #ccc`do we want to change those as well @ericronne  ?

**Comment by ericronne** [comment]
Good catch. Sure.

**Comment by line47** [comment]
Updated! 
screen shot 2018-02-05 at 10 25 19 am
screen shot 2018-02-05 at 10 27 37 am

**Comment by ericronne** [comment]
This icon should go away; if it's not a quick change we can do a sep issue...
image

**Comment by line47** [comment]
Let's do a separate issue addressing the page headers as we are veering outside this body of work. Mind adding a new issue @ericronne ?

File: frontend/sass/_forms.scss [hunk 1]
```diff
@@ -34,13 +35,20 @@ label {
...
-// used to fix the padding on the advanced settings form section 
+// used to fix the padding on the advanced settings form section
 .usa-accordion-content {
   padding: 1.5rem 0;
 }
+
+input,
+textarea,
+select {
+  border: 3px solid #ccc;
```
Comment (jseppi) [inline frontend/sass/_forms.scss]: should `#ccc` be in a variable in `_variables.scss`?

File: frontend/sass/_template-items.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      margin-top: .5em;
+    }
+  }
+
+  .federalist-template-list-item-img {
+    flex-grow: 2;
+  }
+
+  .thumbnail {
+    border: 1px solid #efefef;
```
Comment (jseppi) [inline frontend/sass/_template-items.scss]: color in variable?

File: frontend/components/AddSite/AddRepoSiteForm.jsx [hunk 3]
```diff
@@ -47,14 +46,7 @@ export const AddRepoSiteForm = ({
...
       <GitHubRepoUrlField
-        label="GitHub Repository URL"
-        help={
-          <span>
-            Paste your repository&apos;s GitHub URL here.
-            <br />
-            For example: https://github.com/18f/federalist-docs
-          </span>
-        }
+        label="Already have a GitHub repo for your site? Paste the URL here."
```
Comment (jseppi) [inline frontend/components/AddSite/AddRepoSiteForm.jsx]: In error-state, the field+label kinda jump a little down the page:

error-jump

File: frontend/sass/_forms.scss (hunk 1 above)
Comment (line47) [reply frontend/sass/_forms.scss]: Yeah it should - @ericronne do you have a name for these variables?

File: frontend/components/AddSite/AddRepoSiteForm.jsx (hunk 3 above)
Comment (line47) [reply frontend/components/AddSite/AddRepoSiteForm.jsx]: Fixed the jump here 🐸

**Pull request merged**